Search Feature Format Added

joshualevine opened the pull request:
search page format set up

joshualevine committed:
Add files via upload

search page format set up

neal-merkl commented on HPEMobileApp/app/search/search-page.xml:
```diff
@@ -1,65 +1,101 @@
+                <Label class="search-title" row="0" text="Please fill one or more field:" />
+                <GridLayout class="search-item" row="1" columns="auto, *" >
+                    <Label class="search-field-label" text="System Name:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ system_ID }}" hint="system name" col="1" />
```
`hint` is something you use for additional explanation of what a text field is for.
Having the hint be the same text as the Label defeats the whole point of it.
This should be an example of 'System Name' instead.

This applies to all hints on this page.

neal-merkl commented on HPEMobileApp/app/search/search-page.xml:
```diff
@@ -1,65 +1,101 @@
+                    <TextField class="page-search-field" text="{{ osVersion }}" hint="e.g. '3.2.2.530'" col="1" />
+                </GridLayout>
+                <GridLayout class="search-item" row="7" columns="auto, 2*, auto, *" >
+                    <Label class="search-field-label" text="Location: Region:" col="0" /> 
```
This Label is a little confusing. Either split it up into two different Labels with better spacing or just pick one of the fields to use.

neal-merkl commented on HPEMobileApp/app/search/search-page.xml:
```diff
@@ -1,65 +1,101 @@
+                    <Label class="search-field-label" text="Location: Region:" col="0" /> 
+                    <TextField class="page-search-field" text="{{ region }}" hint="" col="1" />
+                    <Label class="search-field-label" text="Country:" col="2" /> 
+                    <TextField class="page-search-field" text="{{ country }}" hint="" col="3" />
```
Put this on its own line and indent it instead of having it on one line. It gets cluttered on mobile screens.

neal-merkl reviewed (changes requested):
Some layout changes

neal-merkl reviewed (changes requested):
the configuration folder should be inside the HPEMobileApp folder. move the folder and update all file names.

Also it doesn't look like the configuration page is reachable from anywhere?

Also you have code for the menu app drawer inside here, but it looks like that already exists in master. Please verify

joshualevine committed:
Adjusted format to meet suggestions made by anmkerl

joshualevine commented:
Accidentally pushed from old account. Deleted the old account but I can't figure out how to remove the commit. Probably doesn't matter. 
Also, I don't know what you are referring to by "configuration fold" and "configuration page". All I have done is edited the files in the "search" directory and the added the css file for it and added a new directory "search-results" and all the files in it.

neal-merkl commented:
oh, I was reviewing two pull requests at the same time and put comments from the other PR into this one. Sorry, I'll re-review now.

neal-merkl reviewed (approved):
lgtm
